pkg/storkctl/action.go

```diff
@@ -0,0 +1,88 @@
+package storkctl
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	storkops "github.com/portworx/sched-ops/k8s/stork"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/kubectl/pkg/cmd/util"
+)
+
+const (
+	failoverCommand             = "failover"
+	nameTimeSuffixFormat string = "2006-01-02-150405"
+)
+
+func newPerformCommand(cmdFactory Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
+	performCommands := &cobra.Command{
+		Use:   "perform",
+		Short: "Perform actions",
+	}
+
+	performCommands.AddCommand(
+		newFailoverCommand(cmdFactory, ioStreams),
+	)
+	return performCommands
+}
+
+func newFailoverCommand(cmdFactory Factory, ioStreams genericclioptions.IOStreams) *cobra.Command {
+	getClusterPairCommand := &cobra.Command{
```

Comment:
nit: getFailoverCommand

Reply:
The convention used in the code is `newXXXCommand` everywhere, that's why I went with this